packages/shared/package.json

```diff
@@ -5,6 +5,7 @@
   "license": "AGPL-3.0",
   "scripts": {
     "lint:css": "stylelint '**/*.{ts,tsx,css}'",
+    "lint:fix": "eslint . --ext .js,.jsx,.ts,.tsx --fix --max-warnings 0",
```

Comment:
@sshanzel @rebelchris This seemed like a usefull thing to add ^_^

Reply:
Thank you so much for setting this up! I will definitely be one to immediately use this 💯
If you check some previous PRs, you will see a lot having numerous `fix: lint issue` commits. I am guilty as charged 🤣